---
title: LOCALDB_ERROR_INTERNAL_ERROR | Документация Майкрософт
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 60c7a47b-114c-441b-869e-9d8ebfe4148e
author: stevestein
ms.author: sstein
ms.openlocfilehash: 7437a0304192267b510afbdcc33041991673eff7
ms.sourcegitcommit: da88320c474c1c9124574f90d549c50ee3387b4c
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 07/01/2020
ms.locfileid: "85641694"
---
# <a name="localdb_error_internal_error"></a>LOCALDB_ERROR_INTERNAL_ERROR
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
    
## <a name="details"></a>Сведения  
  
|||  
|-|-|  
|Название продукта|SQL Server|  
|Идентификатор события|264|  
|Источник события|Среда выполнения локальной базы данных SQL Server 12.0|  
|Компонент|API среды выполнения локальной базы данных|  
|Текст сообщения|Произошла непредвиденная ошибка внутри вызова метода API экземпляра локальной базы данных. Сведения об ошибке см. в журнале приложений Windows.|  
  
## <a name="explanation"></a>Объяснение  
 Произошла непредвиденная ошибка.  
  
## <a name="user-action"></a>Действие пользователя  
 Подробные сведения см. в журнале событий.  
  
  
